dedupe/api.py

```diff
@@ -987,7 +987,7 @@ class ActiveMatching(Matching):
     """
     Class for training a matcher.
     """
-    classifier = rlr.RegularizedLogisticRegression()
+    classifier = sklearn.ensemble.RandomForestClassifier()
```

Comment:
IDK why someone might have mutliple ActiveMatching instances at the same time, but if so they would share the same classifier. Ideally this should be an instance attribute

Reply:
that's a good call, but an API change. I'll open an issue for that and do that in another PR.